### PR TITLE
add Cybertopia themes

### PR DESCRIPTION
**Pull request by noraj**
New theme with 4 variants, source here (with screenshots): https://github.com/noraj/cybertopia-highlightjs

### CODE REVIEW

File: src/styles/cybertopia-cherry.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+pre code.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 1em
+}
+code.hljs {
+  padding: 3px 5px
+}
```
Comment (joshgoebel) [inline src/styles/cybertopia-cherry.css:28]: Please exclude this, our build system will add the base styling automatically.

**Review by joshgoebel — CHANGES_REQUESTED** [review]
see comments

**Review by Theghost256 — APPROVED** [review]

**Comment by joshgoebel** [comment]
I almost forgot about `checkTheme.js`... please run it and add the few missing styles to your theme.  You can ignore the optional ones if you really want, but we'd like to see all the non-optional ones to include new themes here.

I'll also try to find the time to fix the accessibiltiy checks to work with variables - since we also care about the accessibility of our themes out of the box.

**Comment by noraj** [comment]
Great suggestion, I'll run [checkTheme.js](https://github.com/highlightjs/highlight.js/blob/main/tools/checkTheme.js) as asked in [theme-guide](https://github.com/highlightjs/highlight.js/blob/2bd6ff65635640c99b04cd4217cfe45ca682f884/docs/theme-guide.rst#L52) and add the missing styles.

For accessibility, I checked the contrast against WCAG at https://webaim.org/resources/contrastchecker/.

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
Could you please rebase to resolve conflicts?

**Review by joshgoebel — APPROVED** [review]

**Comment by noraj** [comment]
Thank you for the guidance.

**Pull request merged**